packages/core-data/README.md

```diff
@@ -175,6 +175,7 @@ _Returns_
 
 -   `Object`: Updated record.
 
+
```

Comment:
Super minor, but we can probably omit the tiny changes here.

Reply:
I had the same in another PR where these lines were inserted automatically by the post commit hook. I committed and pushed thinking it was a legitimate README update, but it caused CI to fail. I think this should be reverted as the same will happen here.

Haven't seen the same issue since, but worth keeping an eye out for it..